documentation/known-bugs.md

```diff
@@ -35,7 +35,12 @@ Further investigation might be required to see if there's a better way to change
 3. When clicking on the link to access our account after signing up as a mentor, we are not routed to the right page
 4. We should refresh the mentor's suggested mentees when (s)he creates an account, otherwise they have to log in and out for that to happen
 
-
 March update:
 [JP]
-- There is an infinite loop somewhere in the code, and it fills the DB with info and crashes the front-end. It adds hundreds of times the same email address to the pendingMentors field of the DB.
\ No newline at end of file
+
+- There is an infinite loop somewhere in the code, and it fills the DB with info and crashes the front-end. It adds hundreds of times the same email address to the pendingMentors field of the DB.
+
+[Katrina]
+
+- During the client meeting, I noticied that a TypeError displaying 'Cannot set property 'id' of null' occurs when navigating to the 'Accepted Mentees' tab.
+- The 'Your Suggested Mentees have been updated' notification is displayed for both Mentor and Admin users upon login (the client mentioned that we could likely remove this notification altogether).
```

Comment:
I forgot to add this into the known-bugs.md after the client meeting, and since it had already been reviewed, I didn't want to push this to that branch. I was hoping to fix these myself when I arrived to those components, but I didn't have time. I'm not sure if I'll be able to circle back to this before the sprint ends, so these could be issues for the next sprint.

Reply:
Noted.